src/internal_events/mod.rs

```diff
@@ -25,7 +25,7 @@ pub(crate) mod aws_s3;
 pub(crate) mod aws_s3_sink;
 #[cfg(feature = "sinks-aws_sqs")]
 mod aws_sqs;
-#[cfg(feature = "sinks-azure_blob")]
+#[cfg(any(feature = "sinks-azure_blob", feature = "sinks-datadog_archives"))]
```

Comment:
No need for the change here, since you're including `sinks-azure_blob` in the flags set by `sinks-datadog_archive` already.